rclcpp/src/rclcpp/parameter_client.cpp

```diff
@@ -256,6 +256,15 @@ SyncParametersClient::get_parameters(const std::vector<std::string> & parameter_
   return std::vector<rclcpp::parameter::ParameterVariant>();
 }
 
+bool
+SyncParametersClient::has_parameter(const std::string & parameter_name)
+{
+  std::vector<std::string> names;
+  names.push_back(parameter_name);
+  auto vars = get_parameters(names);
```

Comment:
Getting the parameter value only to check if the parameter exists if wasteful. Consider a parameter which stores a large amount of data.